packages/liquidsoap-daemon/liquidsoap-daemon.0.3.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "romain.beauxis@gmail.com"
+homepage: "https://github.com/savonet/liquidsoap-daemon"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+available: [ os != "win32" ]
+install: [
+  ["./daemonize-liquidsoap.sh"] { os = "linux" }
```

Comment:
this script is called `daemonize-liquidsoap` -- is it really meant to be in the install field?